portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,65 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="home">
+        <h1>Jimin Byun</h1>
+        <div id="links">
+            <a href="https://www.linkedin.com/in/jimin-byun/">LinkedIn</a>
+            <a href="https://github.com/jiminbyun">Github</a>
+        </div>
+    </div>
+    <div id="intro">
+        <h2>Hello! I am Jimin.</h2>
+        <p> I'm 19, rising junior at Carnegie Mellon University, School of Computer Science. 
+            I am majoring in Artificial Intelligence with additional major in Computer Science. 
+            While I enjoyed every new things I learned in school, some of best courses for me was Computer Vision, Planning Teachniques for Robotics and Parallel Algorithms. 
+            I am originally from Seoul, South Korea, although I spent most of my post elementary school years in small island called Jeju. 
+            I came to United States for the first time as a college freshman, so my experience and knowledge about US is limited to Pittsburgh, specifically within 10km radius around our school. 
+        </p>
+
+        <p> In my free time, I enjoy solving jigsaw puzzles while marathoning a movie or tv series. 
+            My friends and I spend most of our time together playing board games or going to Room Escapes. 
+            Contrary to my other hobbies, I also play drum kit for fun, especially for upbeat songs. (Not that I'm good at it)
+        </p>
+    </div>
+    <div id="random">
+        <h2>Random-Random Information</h2>
+        <p>Click here to get a random</p>
+        <button onclick="addRandomGreeting()">...</button>
+        <div id="greeting-container"></div>
+    </div>
+    <div id="exp">
+        <h2>Experiences</h2>
+        <div id="ta">
+            <h3>Teaching Assistant, Spring 2020-Present </h3>
+            <p>Carnegie Mellon, Concepts of A.I, A.I Representation and Problem Solving</p>
+            <ul>
+                <li>Teach various artificial intelligence methods and algorithms to implement them.</li>
```

Comment:
nit: you may omit the closing </li> tag. See https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

Reply:
I tried deleting all the optional tags (including closing tags for body and head). Not sure whether I should delete start tags for heads and body too.